src/cephadm/box/start

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+CEPHADM_PATH=/usr/local/sbin/cephadm
+
+dnf install which sudo -y
+
+cp /cephadm/cephadm $CEPHADM_PATH
+chmod +x $CEPHADM_PATH
+
+tail -f /var/log/ceph/cephadm.log 1>&2 &
+
+EXTRA_ARGS=()
+if [[ -n "${SHARED_CEPH_FOLDER-}" ]]; then
+    EXTRA_ARGS+=(--shared_ceph_folder "$SHARED_CEPH_FOLDER")
+fi
+
+# TODO: remove docker build image and skill pull when cephadm's dependencies
+# use which or it is removed.
+# If we use a ceph image cephadm won't skip pulling the image. If it's a
+# local image, it will fail.
+docker build -t quay.ceph.io/ceph-ci/ceph:master /cephadm/box/docker/ceph
+CEPHADM_IMAGE=quay.ceph.io/ceph-ci/ceph:master
+if [[ -n "$CEPHADM_IMAGE" ]]; then
+	EXTRA_ARGS+=--skip-pull
+fi
+
+$CEPHADM_PATH bootstrap \
+  --mon-ip "$(hostname -i)" \
+  --allow-fqdn-hostname \
+  --initial-dashboard-password admin \
+  --dashboard-password-noupdate \
+  --allow-overwrite \
```

Comment:
please remove this line

Reply:
sure, I don't remember why I added that.